### PR TITLE
Write the project html code

### PR DESCRIPTION
**Pull request by przemekjuzwiszin**

### CODE REVIEW

File: src/content/index.html [hunk 1]
```diff
@@ -33,8 +35,160 @@
...
+              <a class="project__link" href="#" title="Demo: projekt">Demo</a>
+              <a class="project__link" href="#" title="Source code: projekt"
+                >Github</a
+              >
+            </div>
+          </li>
+          <li class="project">
+            <img class="project__logo" src="#" alt="" />
+            <h3 class="project__title">Projekt</h3>
+            <p class="project__description">Opis projektu</p>
```
Comment (robson60000) [inline src/content/index.html]: Everything on this page is in English

File: src/content/index.html [hunk 2]
```diff
@@ -33,8 +35,160 @@
...
+            <div class="project__footer">
+              <a class="project__link" href="#" title="Demo: projekt">Demo</a>
```
Comment (robson60000) [inline src/content/index.html]: Do you want to add  empty links?

**Review by robson60000 — APPROVED** [review]
Technically, i do not observe any bug in your code, but some parts are in EN and some aare in PL.
There are also lots of links but all points to this page instead of e.g. facebook profile

File: src/content/index.html (hunk 2 above)
Comment (przemekjuzwiszin) [reply src/content/index.html]: Yes I wan't to add empty links.

File: src/content/index.html (hunk 1 above)
Comment (przemekjuzwiszin) [reply src/content/index.html]: I corrected the polish words to English.

**Review by robson60000 — APPROVED** [review]

**Pull request merged**